src/lib/xmlParse/xmlQueryContextResponse.cpp

```diff
@@ -374,6 +374,5 @@ void qcrsPresent(ParseData* parseDataP)
   if (!lmTraceIsSet(LmtDump))
     return;
 
```

Comment:
This piece of code here responds part of a question you had about LM_F.

LM_F is always printed, but in the 'present()' methods I dont continue unless the traceLevel LmtDump is set, so it is some kind of hybrid.